src/widget/OktaSignIn.js

```diff
@@ -36,18 +36,19 @@ var OktaSignIn = (function () {
         Router = require('v2/WidgetRouter');
       }
       if (widgetOptions.stateToken) {
+        router = bootstrapRouter.call(this, Router, authClient, widgetOptions, renderOptions, successFn, errorFn);
+        if (!widgetOptions.apiVersion) {
+          widgetOptions.apiVersion = router.settings.get('apiVersion');
```

Comment:
`widgetOptions` already has `apiVersion`.
`router.settings` is actually initialized from `widgetOptions`

Reply:
If we missed specifying the apiVersion , then we want to get the default `1.0.0` from settings and I am setting widgetOptions.apiVersion so that we can call idx-js with `1.0.0`

Reply:
oh I see.. 🤦

Reply:
I did this to avoid breaking the local widget for everyone when they pull this change in as they might not have apiVersion set locally 😃